include/z64actor.h

```diff
@@ -212,15 +212,23 @@ typedef enum {
  * 0x200 : Only set/used by player, related to interacting with walls
  */
 
+#define DYNA_TRANSFORM_POS (1 << 0) // Position of the actors on top follows the dynapoly actor's movement.
+#define DYNA_TRANSFORM_ROT_Y (1 << 1) // The Y rotation of the actors on top follows the dynapoly actor's Y rotation.
+
+#define DYNA_INTERACT_ACTOR_ON_TOP (1 << 0) // There is an actor standing on the collision of the dynapoly actor
+#define DYNA_INTERACT_PLAYER_ON_TOP (1 << 1) // The player actor is standing on the collision of the dynapoly actor
+#define DYNA_INTERACT_PLAYER_ABOVE (1 << 2) // The player is directly above the collision of the dynapoly actor (any distance above)
+#define DYNA_INTERACT_ACTOR_ON_SWITCH (1 << 3) // Like the ACTOR_ON_TOP flag but only actors with ACTOR_FLAG_CAN_HOLD_SWITCH
+#define DYNA_INTERACT_ACTOR_ON_HEAVY_SWITCH (1 << 4) // Like the ACTOR_ON_TOP flag but only actors with ACTOR_FLAG_CAN_HOLD_HEAVY_SWITCH
```

Comment:
Perhaps we should we add a "dyna_actor.h" for these flags and structs?

Reply:
Maybe? I don't think the terminology is fully resolved on dynaActor/bgActor, and they're connected so would this header contain both?

Reply:
Not sure, perhaps it would be best to wait until the terminology is better resolved.